cmake/flash/CMakeLists.txt

```diff
@@ -168,6 +168,8 @@ foreach(target flash debug debugserver attach)
     endif()
   elseif(target STREQUAL attach)
     set(comment "Debugging ${BOARD}")
+  elseif(target STREQUAL rtt)
+    set(comment "Debugging ${BOARD}")
```

Comment:
should this have something other than debugging? maybe just `RTT ${BOARD}`

Reply:
Fixed!